cli/cmd/inject.go

```diff
@@ -93,6 +95,10 @@ sub-folders, or coming from stdin.`,
 	}
 
 	flags := options.flagSet(pflag.ExitOnError)
+	flags.BoolVar(
+		&advancedOption, "advanced", advancedOption,
+		"Add the proxy container in the ouput (as opposed to just adding the annotations for the auto-injector to do it) (default false)",
```

Comment:
typo: _ouput_

Can we include the implication of using this flag (i.e. the proxy injector will skip the YAML output, hence, auto-injection and config override won't work)? How about _Include the proxy sidecar container spec in the YAML output. Note that proxy configuration annotations aren't supported._?

Reply:
I think it's important to mention the auto-injector, so we could put something like
```
Include the proxy sidecar container spec in the YAML output (the auto-injector won't pick it up, so config annotations aren't supported) (default false)
```
A bit long still...

Reply:
Or we can just create a ticket in [website](https://github.com/linkerd/website) to document it.